config/environments/development.rb

```diff
@@ -1,4 +1,12 @@
 Rails.application.configure do
+  #Mailer Previews
+  config.action_mailer.preview_path = Saas::Engine.root.join('test/mailers')
```

Comment:
This line points the `Action::Mailer` previewer to looks for mailer previews in the `Saas::Engine`. Otherwise when we attempt to use previews Rails is not able to find them.